common/lib/xmodule/xmodule/library_content_module.py

```diff
@@ -8,6 +8,7 @@
 import random
 from copy import copy
 from gettext import ngettext
+from typing import Any
```

Comment:
I don't think we're using this.

Reply:
This was most likely a vscode autocompletion thing that I didn't catch. Thanks!